spec/datetimeformat.html

```diff
@@ -265,19 +278,72 @@
-            1. Let _fv_ be an implementation and locale dependent String value representing "ante meridiem".
-          1. Replace the substring of _result_ that consists of *"{ampm}"*, with _fv_.
+            1. Add new part record { [[type]]: _p_, [[value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal "ampm", then:
+            1. Let _v_ be the value of _tm_.[[hour]].
+            1. If _v_ is greater than 11, then:
+              1. Let _fv_ be an implementation and locale dependent String value representing "post meridiem";
+            1. Else,
+              1. Let _fv_ be an implementation and locale dependent String value representing "ante meridiem".
+            1. Add new part record { [[type]]: *"dayPeriod"*, [[value]]: _fv_ } as a new element of the list _result_.
+            1. Let _literal_ be the substring of _pattern_ from position _beginIndex_, inclusive, to position _endIndex_, inclusive.
+            1. Add new part record { [[type]]: *"literal"*, [[value]]: _literal_ } as a new element of the list _result_.
+          1. Set _nextIndex_ to _endIndex_ + 1.
+          1. Set _beginIndex_ to Call(*%StringProto_indexOf%*, _pattern_, "}", _nextIndex_)
+        1. If _nextIndex_ is less than _length_, then:
+          1. Let _literal_ be the substring of _pattern_ from position _nextIndex_, inclusive, to position _length_, exclusive.
+          1. Add new part record { [[type]]: *"literal"*, [[value]]: _literal_ } as a new element of the list _result_.
         1. Return _result_.
       </emu-alg>
 
       <emu-note>
         It is recommended that implementations use the locale and calendar dependent strings provided by the Common Locale Data Repository (available at <a href="http://cldr.unicode.org/">http://cldr.unicode.org/</a>), and use CLDR "abbreviated" strings for DateTimeFormat "short" strings, and CLDR "wide" strings for DateTimeFormat "long" strings.
       </emu-note>
+
+      <emu-note>
+        It is recommended that implementations use the time zone information of the IANA Time Zone Database.
+      </emu-note>
+
+    </emu-clause>
+
+    <emu-clause id="sec-formatdatetime" aoid="FormatDateTime">
+      <h1>FormatDateTime(dateTimeFormat, x)</h1>
+
+      <p>
+        The FormatDateTime abstract operation is called with arguments _dateTimeFormat_ (which must be an object initialized as a DateTimeFormat) and _x_ (which must be a Number value), and performs the following steps:
+      </p>
+
+      <emu-alg>
+        1. Let _parts_ be ? PartitionDateTimePattern(_dateTimeFormat_, _x_).
+        1. Let _result_ be an empty String.
+        1. For each _part_ in _parts_, do:
+          1. Set _result_ to a String value produced by concatenating _result_ and _part_.[[value]].
+        1. Return _result_.
+      </emu-alg>
+
+    </emu-clause>
+
+    <emu-clause id="sec-formattopartsdatetime" aoid="FormatToPartsDateTime">
+      <h1>FormatToPartsDateTime(dateTimeFormat, x)</h1>
```

Comment:
The function it is a counter part to is "FormatDateTime" (Fn+Object), so I'm sticking to match to it.